test/crawlers/cheerio_crawler.test.js

```diff
@@ -629,49 +638,51 @@ describe('CheerioCrawler', () => {
             await crawler.run();
             delete process.env[ENV_VARS.PROXY_PASSWORD];
 
+            utilsRequestMock.verify();
```

Comment:
I think you need `.restore()` here. It both restores and verifies.

Reply:
Changed to `.restore()`.